content/docs/command-reference/exp/show.md

```diff
@@ -78,36 +76,12 @@ metric or param.
 - `--param-deps` - include only parameters that are stage dependencies.
 
 - `--only-changed` - show only parameters and metrics with values that vary
-  across experiments. Note that this option takes precedence over
-  `--include-params` and `--include-metrics`, for example given
-  `--include-params=foo --only-changed`, param `foo` would still be hidden if
-  its value is the same in all experiments.
-
-- `--include-params <list>` - show the specified `dvc params` in the table only.
-  Accepts a comma-separated `list` of param names. Shell style wildcards
-  supported: `*`, `?`, `[seq]`, `[!seq]`, and `**` If a `path:` prefix is
-  included in the name, the wildcard pattern will **not** be matched against
-  that prefix but only against the rest of the name.
-
-- `--exclude-params <list>` - hide the specified `dvc params` from the table
-  (all param will be shown except for these). Accepts a comma-separated `list`
-  of param names. Shell style wildcards supported: `*`, `?`, `[seq]`, `[!seq]`,
-  and `**` If a `path:` prefix is included in the name, the wildcard pattern
-  will **not** be matched against that prefix but only against the rest of the
-  name.
-
-- `--include-metrics <list>` - show the specified `dvc metrics` in the table
-  only. Accepts a comma-separated `list` of metric names. Shell style wildcards
-  supported: `*`, `?`, `[seq]`, `[!seq]`, and `**`. If a `path:` prefix is
-  included in the name, the wildcard pattern will **not** be matched against
-  that prefix but only against the rest of the name.
-
-- `--exclude-metrics <list>` - hide the specified `dvc metrics` from the table
-  (all param will be shown except for these). Accepts a comma-separated `list`
-  of metric names. Shell style wildcards supported: `*`, `?`, `[seq]`, `[!seq]`,
-  and `**`. If a `path:` prefix is included in the name, the wildcard pattern
-  will **not** be matched against that prefix but only against the rest of the
-  name.
+  across experiments.
+
+- `--drop <regex>` - remove the columns matching the specified [regex][regex].
+
+- `--keep <regex>` - prevent the columns matching the specified [regex][regex]
+  to be removed by any of the other options.
```

Comment:
I hope we can introduce them without the concept of regex at first, and then mention separately any regex is accepted.

Reply:
I'm not sure what term to use here instead of regex.

Reply:
"Pattern"? "Name"? Or simply "remove the matching columns."

Reply:
```suggestion
- `--drop <regex>` - remove the matching columns.

- `--keep <regex>` - prevent the matching columns to be removed by any of the other options.
```